sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -557,12 +648,47 @@ export class EventHubReceiver extends LinkEntity {
     );
   }
 
+  private async _useInternalQueue(
+    onMessage: OnMessage,
+    abortSignal?: AbortSignalLike
+  ): Promise<number> {
+    let processedMessages = 0;
```

Comment:
Rename to `processedCount` or `processedMessagesCount` to avoid the confusion that this variable tracks the actual messages